src/Compilers/CSharp/Portable/Generated/CSharp.Generated.g4

```diff
@@ -0,0 +1,1218 @@
+  | 'true'
+  | character_literal_token
+  | numeric_literal_token
+  | string_literal_token
+  ;
+
+make_ref_expression
+  : '__makeref' '(' expression ')'
+  ;
+
+member_access_expression
+  : expression ('->' | '.') simple_name
+  ;
+
+member_binding_expression
+  : '.' simple_name
+  ;
+
+object_creation_expression
+  : 'new' type argument_list? initializer_expression?
+  ;
+
+omitted_array_size_expression
+  : /* epsilon */
+  ;
+
+parenthesized_expression
+  : '(' expression ')'
+  ;
+
+postfix_unary_expression
+  : expression ('!' | '++' | '--')
+  ;
+
+prefix_unary_expression
+  : ('!' | '&' | '*' | '+' | '++' | '-' | '--' | '^' | '~') expression
+  ;
+
+query_expression
+  : from_clause query_body
+  ;
+
+from_clause
+  : 'from' type? identifier_token 'in' expression
+  ;
+
+query_body
+  : query_clause+ select_or_group_clause query_continuation?
+  ;
+
+query_clause
+  : from_clause
+  | join_clause
+  | let_clause
+  | order_by_clause
+  | where_clause
+  ;
+
+join_clause
+  : 'join' type? identifier_token 'in' expression 'on' expression 'equals' expression join_into_clause?
+  ;
+
+join_into_clause
+  : 'into' identifier_token
+  ;
+
+let_clause
+  : 'let' identifier_token '=' expression
+  ;
+
+order_by_clause
+  : 'orderby' ordering (',' ordering)*
+  ;
+
+ordering
+  : expression ('ascending' | 'descending')?
+  ;
+
+where_clause
+  : 'where' expression
+  ;
+
+select_or_group_clause
+  : group_clause
+  | select_clause
+  ;
+
+group_clause
+  : 'group' expression 'by' expression
+  ;
+
+select_clause
+  : 'select' expression
+  ;
+
+query_continuation
+  : 'into' identifier_token query_body
+  ;
+
+range_expression
+  : expression? '..' expression?
+  ;
+
+ref_expression
+  : 'ref' expression
+  ;
+
+ref_type_expression
+  : '__reftype' '(' expression ')'
+  ;
+
+ref_value_expression
+  : '__refvalue' '(' expression ',' type ')'
+  ;
+
+size_of_expression
+  : 'sizeof' '(' type ')'
+  ;
+
+stack_alloc_array_creation_expression
+  : 'stackalloc' type initializer_expression?
+  ;
+
+switch_expression
+  : expression 'switch' '{' (switch_expression_arm (',' switch_expression_arm)* ','?)? '}'
+  ;
+
+switch_expression_arm
+  : pattern when_clause? '=>' expression
+  ;
+
+throw_expression
+  : 'throw' expression
+  ;
+
+tuple_expression
+  : '(' argument (',' argument)* ')'
+  ;
+
+type_of_expression
+  : 'typeof' '(' type ')'
+  ;
+
+type
+  : array_type
+  | name
+  | nullable_type
+  | omitted_type_argument
+  | pointer_type
+  | predefined_type
+  | ref_type
+  | tuple_type
+  ;
+
+nullable_type
+  : type '?'
+  ;
+
+omitted_type_argument
+  : /* epsilon */
+  ;
+
+pointer_type
+  : type '*'
+  ;
+
+predefined_type
+  : 'bool'
+  | 'byte'
+  | 'char'
+  | 'decimal'
+  | 'double'
+  | 'float'
+  | 'int'
+  | 'long'
+  | 'object'
+  | 'sbyte'
+  | 'short'
+  | 'string'
+  | 'uint'
+  | 'ulong'
+  | 'ushort'
+  | 'void'
+  ;
+
+ref_type
+  : 'ref' 'readonly'? type
+  ;
+
+tuple_type
+  : '(' tuple_element (',' tuple_element)* ')'
```

Comment:
>* [](start = 41, length = 1)

The file probably needs to be re-generated

Reply:
why?